docs/build/sandboxes.md

```diff
@@ -0,0 +1,196 @@
+under the Identity section of the Setup tab (right at the top of the page).
+
+## Merging sandboxes
+
+Once you've finished making changes to your workflows, it's time to merge them
+back into your main project.
+
+This is easy in the app: simply head to the Sandboxes page, find the Sandbox you
+want to merge in the list, and click the Merge icon on the right-hand side.
+
+You'll be prompted to select the target project or sandbox to merge into: pick
+from the list and click Merge. Usually you'll want to merge into the original
+project, which is selected by default.
+
+When merging, we replace the contents of workflows in your project with those in
+your sandbox. Any workflows which are not in the sandbox will be ignored. If you
+rename a workflow in the sandbox, you'll see the new workflow appear in your
+main project, and the original workflow will be left alone (you'll probably want
+to delete that manually).
+
+Note that settings and options, like concurrency and data retention rules, are
+not transferred in the merge, nor are historical runs or dataclips. Just the
+Workflow contents.
+
+After merging, the sandbox will be destroyed, along with its history and
+dataclips.
+
+:::tip
+
+You can also use the CLI to merge your changes locally, give them one final
+test, and then deploy them to your main project.
+
+::::
+
+## Conflicts
+
+If you've ever worked with a source version control system - like git or
+Subversion - you'll be familiar with the idea of conflicts.
+
+A conflict occurs when you're trying to merge a Sandbox into your original
+project (or another sandbox), and there are incompatible changes between them.
+
+Say you create a Sandbox from your main project and in the sandbox, you change
+the adaptor of one step from `common` to `http`. And while you're making this
+change, a colleague goes to the main project and sets the adaptor of the same
+step to `salesforce`.
+
+What happens when you try and merge the sandbox? Should we preserve the original
+change? Or accept the change in the sandbox? Or something else?
+
+Occasionally these conflicts are trivial to resolve and you might wonder what
+all the fuss is about. But often they are complex, and it can be difficult or
+impossible to automate a solution.
+
+[TODO not implemented yet] When we detect a conflict like this, we'll show a
+warning when you try and merge the Sandbox. You can choose to "force push" the
+Sandbox and overwrite whatever changes happened on the target Project, or you
+can cancel and resolve the conflict yourself.
+
+For now, the only way to resolve conflicts manually is to use the CLI to edit
+your project locally, and pushed the resolved, final version up to the app.
+
+:::tip
+
+We'll be adding better support for resolving conflicts soon.
+
+:::
+
+## Editing sandboxes Locally
+
+:::info
+
+Sandboxes are fully compatible with the CLI.
+
+For more details, see these CLI docs (link to follow)
```

Comment:
Ooh okay, this make sense maybe we should add the link also to this section https://github.com/OpenFn/docs/pull/712/files#r2432034516

Reply:
I'm going to leave it blank for now, but I plan to write up really thorough docs on the CLI side (the CLI isn't really ready yet)